woocommerce/woocommerce-gutenberg-products-block
# New block: Filter Products by Price

Introduces the filter products by price block (#710).

**Screenshots**

![2019-10-30 16 24 27](https://user-images.githubusercontent.com/90977/67877657-e24a4c00-fb31-11e9-987d-003e5c1f5939.gif)

**How to test the changes in this Pull Request:**

1. Insert block on the same page as an all products block
2. Check it filters correctly
3. Text with inputs/buttons hidden. When button is hidden, there should be a debounce/delay after using the filter.

**Changelog**

> Feature - Filter Products by Price block. Allow customers to filter the all products block by price.
